css/build.css

```
@@ -145,6 +145,43 @@
   white-space: nowrap;
 }
 
+.multiple-images-holder {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.multiple-images-item {
+  width: calc(50% - 7.5px);
+  margin-top: 15px;
+  border-radius: 12px;
+  padding: 10px;
+  border: 1px solid rgba(127, 127, 127, 0.5);
+  cursor: pointer;
+}
+
+.multiple-images-item:nth-of-type(2n + 1) {
+  margin-right: 15px;
+}
+
+.multiple-images-item .image {
+  width: 100%;
+  padding-top: 100%;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+@media screen and (min-width: 640px) {
+  .multiple-images-item {
+    width: calc(33.33% - 10px);
```

Why is this 10px when the mobile one is 7.5px?

```suggestion
    width: calc(33.33% - 7.5px);
```